kernel/aster-nix/src/taskless.rs

```diff
@@ -133,10 +133,10 @@ fn do_schedule(
     {
         return;
     }
-
-    CpuLocal::borrow_with(taskless_list, |list| {
-        list.lock_irq_disabled().push_front(taskless.clone());
-    });
+    taskless_list
+        .borrow_irq_disabled()
+        .lock()
+        .push_front(taskless.clone());
```

Comment:
@cchanging Could you refresh my memory as to why we have prefered `CpuLocal<SpinLock<LinkedList<TasklessAdapter>>>` over the more light-weight `CpuLocal<RefCell<LinkedList<TasklessAdapter>>>`?

Reply:
Now I also think it seems better to use `CpuLocal<RefCell<LinkedList<TasklessAdapter>>>` and call `borrow_irq_disabled()`. The requirement here should be to disable irq and achieve interior mutability. Previously, with only the `borrow` interface available, it might have been more convenient to use a `SpinLock` and we may have overlooked this point.

Reply:
Ok. Then we agree on adopting `CpuLocal<RefCell<LinkedList<TasklessAdapter>>>`.

@junyang-zh Could you also fix this point in this PR?